src/scales/scale.time.js

```diff
@@ -80,6 +80,31 @@ function arrayUnique(items) {
 	return out;
 }
 
+/**
+ * @TODO remove options.time.min at v3
+ */
+function getSpecifiedMin(options) {
```

Comment:
How about adding `time.min` and `time.max` there: https://github.com/chartjs/Chart.js/blob/24de2660e86369fbdc22545e13be97fbe2e2422a/src/scales/scale.time.js#L500
and using `valueOrDefault` instead of these new functions?

Reply:
Yeah, good catch. That's a great idea. I left the functions to reduce code duplication, but made the other changes. Thanks for all the reviews today